resources/authgear/static/authgear.css

```diff
@@ -623,7 +615,7 @@ h1, h2, h3, h4, h5, h6 {
   flex-direction: row;
 }
 
-.messages-bar.display-none {
+.messages-bar.hidden {
```

Comment:
I guess this declaration is actually redundant. I made a diff to remove this redundant declaration.

```diff
diff --git a/resources/authgear/static/authgear.css b/resources/authgear/static/authgear.css
index eed4da114..d91728ba0 100644
--- a/resources/authgear/static/authgear.css
+++ b/resources/authgear/static/authgear.css
@@ -610,15 +610,6 @@ h1, h2, h3, h4, h5, h6 {
   width: 100%;
 }
 
-.messages-bar {
-  display: flex;
-  flex-direction: row;
-}
-
-.messages-bar.hidden {
-  display: none;
-}
-
 .messages-bar .icon {
   font-size: 16px;
   min-width: 16px;
diff --git a/resources/authgear/templates/en/web/__error.html b/resources/authgear/templates/en/web/__error.html
index 96f0b7498..f23884a91 100644
--- a/resources/authgear/templates/en/web/__error.html
+++ b/resources/authgear/templates/en/web/__error.html
@@ -13,7 +13,7 @@
         {{ $display_error = false }}
     {{ end }}
 {{ end }}
-<div class="messages-bar errors-messages-bar errors {{ if not $display_error }}hidden{{ end }}" data-network-error="{{ template "error-network" }}" data-server-error="{{ template "error-server" }}">
+<div class="messages-bar flew-row errors-messages-bar errors {{ if not $display_error }}hidden{{ else }}flex{{ end }}" data-network-error="{{ template "error-network" }}" data-server-error="{{ template "error-server" }}">
     <div class="icon"><i class="ti ti-alert-circle"></i></div>
     <ul class="messages-txt error-txt">
         {{ if $display_error }}
diff --git a/resources/authgear/templates/en/web/__message_bar.html b/resources/authgear/templates/en/web/__message_bar.html
index 5877bef26..ffb709f5b 100644
--- a/resources/authgear/templates/en/web/__message_bar.html
+++ b/resources/authgear/templates/en/web/__message_bar.html
@@ -4,7 +4,7 @@
 {{ template "__error.html" . }}
 
 {{ if .FlashMessageType }}
-<div class="messages-bar success">
+<div class="messages-bar flex flex-row success">
     <div class="icon"><i class="ti ti-circle-check"></i></div>
     <ul class="messages-txt">
         {{ if (eq .FlashMessageType "resend_code_success") }}
```